app/models.py

```diff
@@ -1432,6 +1436,9 @@ class NotificationHistory(db.Model, HistoryModel):
     created_by = db.relationship('User')
     created_by_id = db.Column(UUID(as_uuid=True), db.ForeignKey('users.id'), nullable=True)
 
+    postage = db.Column(db.String, nullable=True, default='second')
```

Comment:
There's a default of 'second' for `NotificationHistory` but not for `Notification` - is that deliberately not in `Notification`, or should both models be the same?

Reply:
ah i should have removed this - it shouldn't be on either